Source/src/modules/ModuleRender.h

```diff
@@ -134,6 +134,8 @@ namespace Hachiko
             glDisable(GL_BLEND);
         }
     
+        void RenderDeferredQuad() const;
```

Comment:
Just curious, we can't manage leave this as private and implement the video feature? I suppose we need this public for video?

Reply:
Yes, is for rendering the square where the video frames will be shown (it's called in ComponentVideo::Draw() ). As we talk about particles, I thought it was better to reuse deferred quad instead of creating a new one. 
An option to keep it private could be create a getter for deferred_quad_vao and call Bind and DrawElements inside ComponentVideo. But I think is simpler now. What do you think?